### PR TITLE
Feature/ci setup

### PR DESCRIPTION
**Pull request by StevieAmb**
### 1. What changed?
- A file for continuous integration is created, so that our tests can be run with each push.

### 2. Why is this change necessary?
- So that our tests can be run with each push, and we will not be able to merge without the tests running.

### 3. What changed technically that may impact others?
- The files cannot be merged now if the tests are not passing. 

### 4. How do we test it?
- We click the actions tab in our repo, click into the latest test, the one on top, and look for that green checkmark.

### CODE REVIEW

File: .github/workflows/run-test.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: run-tests
```
Comment (Fordo29) [inline .github/workflows/run-test.yml:1]: Great job on the initial set up

Comment (anewb87) [reply .github/workflows/run-test.yml:1]: Stevie is a wizz. Way to get this moving so quickly

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import logo from './logo.svg';
-import './App.css';
+import './App.scss';
```
Comment (ectweitmann) [inline src/App.js:2]: Nice catch. Thanks for fixing this!

**Review by ectweitmann — APPROVED** [review]

File: src/App.js (hunk 2 above)
Comment (anewb87) [reply src/App.js:2]: My fault! Thanks for fixing.

**Review by anewb87 — APPROVED** [review]
Awesome. Way to set this up so quickly and cleanly.

**Pull request merged**